service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,69 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+{product-title} routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an {product-title} route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio {product-title} Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process, but you must enable it first.
```

Comment:
Istio {product-title} Routing (IOR) <- I think we probably don't want to use a variable here, because the generated output is not going to match the acronym.  (It's going to be Istio OpenShift Container Platform Routing, not Istio OpenShift Routing like we want).

"You can deploy it as part of the control plane in the {ProductName} installation process, but you must enable it first."/ "You enable IOR as part of the control plane deployment."

Reply:
done